tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "target": "esnext",
```

Comment:
A bit late to ask, but wouldnt this make the genrated code to require additional processing?
How about setting it to a released ES version, such as es6?
https://www.typescriptlang.org/tsconfig#target

Reply:
Yes, you're right! This was an oversight on my part and we need to open a pull request to fix this. I think ES5/ES6 is a sensible target to set instead.

Reply:
https://github.com/open-draft/strict-event-emitter/pull/13